app/views/organisations/_featured_news_no10.html.erb

```diff
@@ -17,7 +17,14 @@
       } %>
     </div>
   </div>
-
+  <%
+    @documents.first_featured_news[:context][:text] = "Featured"
```

Comment:
You could theoretically get this from the localisation files, but don't let that be a blocker.